apps/src/accounts/ChangeUserTypeModal.jsx

```diff
@@ -142,7 +142,6 @@ export default class ChangeUserTypeModal extends React.Component {
             onCancel={this.cancel}
             disableConfirm={STATE_SAVING === saveState || !isFormValid}
             disableCancel={STATE_SAVING === saveState}
-            tabIndex="2"
```

Comment:
The Button in the component library is supposed to be accessible, plus it does not support `tabIndex` prop. Is there an additional behaviour associated to `tabIndex` that we would want to preserve?

Reply:
I think as long as it visually looks correct (in comparison to other layers) and it is tab navigable, this is safe to remove!